### DISCUSSION TITLE
TVRename flagged by my corp as possibe keylogger

### DISCUSSION THREAD

**Discussion by chimpsinties** [TV-Rename/tvrename#921] [General]
I downloaded and installed the latest version of TVRename last Friday 17th June 2022 after being informed by the app there was a new release. I used the link in the app notification to download the new version. Upon installing it Windows kicked off a bit saying it was an unverified exe but I carried on because I've been using TVRename for years and trust it. This morning my corporate network admin is complaining saying I ran an application that is a possible keylogger. 
![image](https://user-images.githubusercontent.com/48385801/174566553-ecbce0f3-1f72-4a43-9e6c-5ca0bf0fca21.png)

**Comment by SirSparkles** [comment]
The only 'system hook' I can think of is the fact that TV Rename listens to the filesystem for any new additions to the search folders

**Reply by chimpsinties** [reply]
When I came to install it, there was a message about the exe not being certified or something, can't quite remember now. I bypassed it as I've never had issues with TVRename before. This is Windows endpoint defender that's kicking off. I was a bit surprised how much work knew about what I was doing on another profile of my PC to be honest.
